test/e2e/storage/persistent_volumes-local.go

```diff
@@ -522,7 +540,12 @@ func cleanupLocalVolume(config *localTestConfig, volume *localTestVolume) {
 	if volume.localVolumeType == TmpfsLocalVolumeType {
 		unmountTmpfsLocalVolume(config, volume.hostDir)
 	}
-
+	if volume.localVolumeType == GCELocalSSDVolumeType {
```

Comment:
I would do a similar refactoring here with what you did for the setup.  Have a setup for each type, and a map to point to the proper function.